proposals/forms.py

```diff
@@ -523,10 +523,12 @@ def clean(self):
                     self.add_error(study_name, _('Dit veld is verplicht.'))
 
 
-class ProposalDataManagementForm(forms.ModelForm):
+class ProposalDataManagementForm(SoftValidationMixin, forms.ModelForm):
```

Comment:
If you're going to use SoftValidationMixin, you'll probably want to add the form to ``_build_forms`` in ``proposals/utils/validate_proposal.py`` as well, as otherwise a user won't be stopped properly ;)

Reply:
Derp, you did that. Ignore me